src/CurlRequester.php

```diff
@@ -41,6 +43,14 @@ public function options($options)
             $curl_options[$key] = $options[$value];
         }
 
+        if (!isset($curl_options[CURLOPT_CAINFO])) {
+            $curl_options[CURLOPT_CAINFO] = DEFAULT_CA_CERTS;
+        };
+
+        if ($curl_options[CURLOPT_CAINFO] == "IGNORE") {
```

Comment:
I don't know if this a wellknown thing on php, maybe we could add something in readme about how to set either a different ca cert of set to not use any cert?

Reply:
Yeah I think that's a good idea. There's a lot of stuff missing from the README

Reply:
Turn out we don't call out how to cert in repo. Close this convo